R/get_fy_dates.R

```diff
@@ -7,8 +7,12 @@
 #'
 #' @examples
 #' start_fy("1718")
-start_fy <- function(year) {
-  as.Date(paste0(convert_fyyear_to_year(year), "-04-01"))
+start_fy <- function(year, format = c("fyear", "alternate")) {
+  if (format == "fyyear") {
```

Comment:
Typo in the param specification, also to get R to check the params for you (and throw an error if they don't match) use `match.arg`.
```suggestion
start_fy <- function(year, format = c("fyyear", "alternate")) {
format <- match.arg(format)
  if (format == "fyyear") {
```
If we want a default, use `missing`
```suggestion
start_fy <- function(year, format = c("fyyear", "alternate")) {
if (missing(format)) format <- "fyyear"
format <- match.arg(format)
  if (format == "fyyear") {
```